WhaleSpotting/Models/ResponseModels/SightingResponseModel.cs

```diff
@@ -0,0 +1,53 @@
+using Newtonsoft.Json;
+using System;
+using WhaleSpotting.Models.DbModels;
+
+namespace WhaleSpotting.Models.ApiModels
+{
+    public class SightingResponseModel
+    {
+        [JsonProperty("sighted_at")]
+        public DateTime SightedAt { get; set; }
+        [JsonProperty("species")]
+        public string Species { get; set; }
+        [JsonProperty("quantity")]
+        public int? Quantity { get; set; }
+        [JsonProperty("location")]
+        public string? Location { get; set; }
```

Comment:
strings are nullable without the ?, so you do't need them

Reply:
done